ReplayKitExample/ReplayKitExample/ViewController.swift

```diff
@@ -241,6 +241,14 @@ class ViewController: UIViewController, RPBroadcastActivityViewControllerDelegat
         }
     }
 
+    func room(_ room: TVIRoom, isReconnectingWithError error: Error?) {
+        print("Reconnecting to room \(room.name), error = \(String(describing: error))")
+    }
+
+    func didReconnect(to room: TVIRoom) {
+        print("Reconnected to room \(room.name)")
+    }
```

Comment:
Update SampleHandler with the new callbacks as well.

Reply:
Good catch. Missed that one.

Reply:
Added